src/Board.php

```diff
@@ -84,6 +84,9 @@ class Board implements \ArrayAccess, \Iterator, \JsonSerializable
     /** @var array<int, ?Piece> */
     private $squares = [];
 
+    /** @var bool */
+    private $reversed = false;
```

Comment:
I think, it is time to think about upgrade to php 7.4 :wink: :thinking:

Reply:
I'd love to, but since this is a library I think we should try to keep it as much compatible as possible